### PR TITLE
Add extension for 64k pages on aarch64

### PR DESCRIPTION
**Pull request by jbtrystram**
xref https://issues.redhat.com/browse/COS-2402

### CODE REVIEW

File: extensions-c9s.yaml [hunk 1]
```diff
@@ -74,3 +74,9 @@ extensions:
       - sig-virtualization
     packages:
       - kata-containers
+  # https://issues.redhat.com/browse/OCPSTRAT-649
+  kernel-64k-pages:
```
Comment (cgwalters) [inline extensions-c9s.yaml:78]: Naming bikeshed: is it really worth "renaming" the kernel package here?  i.e. vs just calling it `kernel-64k`

**Review by cgwalters — APPROVED** [review]

**Comment by travier** [comment]
Let's link to the COS epic (https://issues.redhat.com/browse/COS-2402) in the title and commit instead of the START card.

**Comment by travier** [comment]
Can you update the first comment here as well? It's not updated when you push new commits unfortunately :/.

**Comment by travier** [comment]
Looks like we do: https://gitlab.com/redhat/centos-stream/rpms/kernel/-/blob/c9s/kernel.spec?ref_type=heads#L1560
https://composes.stream.centos.org/production/latest-CentOS-Stream/compose/BaseOS/aarch64/os/Packages/

**Comment by jbtrystram** [comment]
> Looks like we do:
> 
>     * [gitlab.com/redhat/centos-stream/rpms/kernel/-/blob/c9s/kernel.spec?ref_type=heads#L1560](https://gitlab.com/redhat/centos-stream/rpms/kernel/-/blob/c9s/kernel.spec?ref_type=heads#L1560)
> 
>     * [composes.stream.centos.org/production/latest-CentOS-Stream/compose/BaseOS/aarch64/os/Packages](https://composes.stream.centos.org/production/latest-CentOS-Stream/compose/BaseOS/aarch64/os/Packages/)
> 
`

Thanks for the pointers !  I am updating this. 
I'll try to add some logic to have the `kernel-debug` extension adding the appropriate packages as well

File: extensions-c9s.yaml [hunk 2]
```diff
@@ -74,3 +74,12 @@ extensions:
...
       - kata-containers
+  # https://issues.redhat.com/browse/COS-2402
+  kernel-64k:
+    architectures:
+      - aarch64
+    packages:
+      - kernel-64k-core
+      - kernel-64k-modules
+      - kernel-64k-modules-core
+      - kernel-64k-modules-extra
```
Comment (travier) [inline extensions-c9s.yaml:85]: Nit: You need an empty line at the end of a file. Make sure to configure your editor to do it for you. In some editors, this empty line is not shown, in some it is.

**Comment by cgwalters** [comment]
> We'll need something like https://github.com/coreos/rpm-ostree/pull/4386 as well for rpm-ostree

Ugh.  Yes.  And we'll need to get that into 4.14.

**Comment by cgwalters** [comment]
https://github.com/coreos/rpm-ostree/pull/4583

**Review by jlebon — APPROVED** [review]
Looks sane to me!

**Comment by cgwalters** [comment]
/lgtm
But this will still need a new rpm-ostree release in practice it sounds like; hopefully soon

**Pull request merged**